spec/controllers/content_items_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe ContentItemsController, type: :controller do
+  describe "GET #index" do
+    let(:organisation) { FactoryGirl.create(:organisation) }
+
+    before do
+      get :index, params: { organisation_id: organisation }
+    end
+
+    it "returns http success" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "assigns content_items" do
```

Comment:
Nice controller testing.

I would ensure that the correct list of content items are assigned as an instance variable. We would suggest going with something like this.

```ruby
    it "assigns content_items" do
      content_items = FactoryGirl.build_list(:content_items, 1)
      allow(organisation).to receive(:content_items).and_return(content_items)

      expect(assigns(:content_items)).to eq(:some_content_items)
    end
```